modules/optimizing-by-encapsulation.adoc

```diff
@@ -1,41 +1,41 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/optimizing-cpu-usage
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
 :_content-type: CONCEPT
 [id="optimizing-cpu-usage_{context}"]
 = Encapsulating mount namespaces
 
 Mount namespaces are used to isolate mount points so that processes in different namespaces cannot view each others' files. Encapsulation is the process of moving Kubernetes mount namespaces to an alternative location where they will not be constantly scanned by the host operating system.
 
-The host operating system uses systemd to constantly scan all mount namespaces: both the standard Linux mounts and the numerous mounts that Kubernetes uses to operate. The current implementation of kubelet and CRI-O both use the top-level namespace for all container runtime and kubelet mountpoints. However, encapsulating these container-specific mountpoints in a private namespace reduces systemd overhead with no difference in functionality. Using a separate mount namespace for both CRI-O and kubelet can encapsulate container-specific mounts from any systemd or other host OS interaction.
+The host operating system uses systemd to constantly scan all mount namespaces: both the standard Linux mounts and the numerous mounts that Kubernetes uses to operate. The current implementation of kubelet and CRI-O both use the top-level namespace for all container runtime and kubelet mount points. However, encapsulating these container-specific mount points in a private namespace reduces systemd overhead with no difference in functionality. Using a separate mount namespace for both CRI-O and kubelet can encapsulate container-specific mounts from any systemd or other host operating system interaction.
```

Comment:
should host operating system be hyphenated? Host-operating-system? You use it frequently enough here as a noun, I'm thinking it could use hyphens.

Reply:
No don't think so. "host operating system" is found in lots of places in the repo.

Reply:
Also: https://redhat-documentation.github.io/supplementary-style-guide/#guest-operating-system